ui.apps/pom.xml

```diff
@@ -101,6 +101,7 @@
                         </goals>
                         <configuration>
                             <nodeVersion>v10.14.2</nodeVersion>
+                            <npmVersion>v6.4.1</npmVersion>
```

Comment:
@christinermurray , I believe this will end up trying to download npm with the following URL from the registry: https://registry.npmjs.org/npm/-/npm-v6.4.1.tgz which is wrong.
Lose the `v` so you have `<npmVersion>6.4.1</npmVersion>` and in case the `install node and npm` exectution is triggered, the URL will be right. :)
Also, is there a reason we force 6.4.1 and not 6.8.0 (current latest).